osgp/protocol-adapter-iec60870/README.md

```diff
@@ -10,3 +10,64 @@ Grid eXchange Fabric detailed documentation:
 
 Grid eXchange Fabric issue tracker:
 * [Grid eXchange Fabric Jira](https://smartsocietyservices.atlassian.net/projects/OC/issues/)
+
+## Add 104 devices to the database
+
+...
+-- osgp_core
+
+INSERT INTO device
+    (creation_time, modification_time, version, device_identification, device_type, is_activated,
+     protocol_info_id, network_address, in_maintenance, gateway_device_id,
+     device_model, device_lifecycle_status)
+  VALUES
+    (current_timestamp, current_timestamp, 0, 'GATEWAY_1', null, true,
```

Comment:
Maybe we could use LMG (or LIGHT_MEASUREMENT_GATEWAY) as devicetype?
I was currently using LMG_1 as identification in my feature branch, but I'll guess GATEWAY_1 is also fine.